subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/CrossVersionPerformanceTestRunner.groovy

```diff
@@ -210,6 +229,13 @@ class CrossVersionPerformanceTestRunner extends PerformanceTestSpec {
         this.buildAction = tapiAction
         return tapiAction
     }
+
+    def setupAndroidStudioSync() {
+        useAndroidStudio = true
+        buildAction = new AndroidStudioSyncAction()
+        studioInstallDir = StudioFinder.findStudioHome()
+        studioJvmArgs = ["-Xms256m", "-Xmx4096m"]
```

Comment:
This should probably be test project specific, WDYT?

Reply:
Jvm args or all parameters? It makes sense for Jvm args yes.